### ISSUE TITLE
`SlevomatCodingStandard.Attributes.AttributesOrder` sniff breaks further file sniffing

### ISSUE THREAD

**Issue by lptn** [slevomat/coding-standard#1457]
First, I'm not sure this is directly connected to this repository (as I have the same issue with `Squiz` ruleset).

My setup: I have a ruleset that extends/config another ruleset that includes all slevomat rules with a lot of excludes like:
```xml
<rule ref="SlevomatCodingStandard">
    <exclude name="SlevomatCodingStandard.Functions.RequireTrailingCommaInCall.MissingTrailingComma"/>
    <exclude name="SlevomatCodingStandard.Files.LineLength.LineTooLong"/>
</rule>
<rule ref="Squiz.Commenting.FunctionComment">
    <exclude name="Squiz.Commenting.FunctionComment.SpacingAfterParamName"/>
</rule>
```

For some reason, after updating from 8.5.0 to 8.6.0 phpcs started to report about failing excluded rules.

composer output on update:
```
Lock file operations: 0 installs, 2 updates, 0 removals
  - Upgrading phpstan/phpdoc-parser (1.8.0 => 1.11.0)
  - Upgrading slevomat/coding-standard (8.5.2 => 8.6.0)
```

**PHP** 8.1.10

Important: all reports for failed files starts from these 2 PHPCS errors:
```
FILE: .../tests/Unit/Notifications/SampleNotificationWithMiddleware.php
------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | [x] Opening PHP tag must be on a line by itself (PSR12.Files.OpenTag.NotAlone)
 1 | ERROR | [ ] An error occurred during processing; checking has been aborted. The error message was: Attributes
   |       |     order not specified. (Internal.Exception)
```
code example (my config of phpcs also reports about it):
```php
<?php declare(strict_types=1);

namespace Tests\Unit\Notifications;

use App\Notifications\MailMessage;
use App\Notifications\MailNotification;
use App\Notifications\ThroughMiddleware;

#[ThroughMiddleware(SampleNotificationMiddleware::class)]
final class SampleNotificationWithMiddleware extends MailNotification
{
    public function build(): MailMessage
    {
        return (new MailMessageBuilder())->build();
    }
}
```

**Comment by kukulich** [comment]
> I have a ruleset that extends/config another ruleset that includes all slevomat rules with a lot of excludes like

This is not ideal setup. You should include only rules you really want.

It's probably caused by some of the new sniffs.

**Comment by lptn** [comment]
@kukulich 
You are correct, I found `SlevomatCodingStandard.Attributes.AttributeAndTargetSpacing` rule caused this

**Comment by lptn** [comment]
@kukulich 
You are right, `SlevomatCodingStandard.Attributes.AttributesOrder` caused this issue

**Comment by kukulich** [comment]
@lptn So both sniffs have problem?

**Comment by lptn** [comment]
@kukulich  `SlevomatCodingStandard.Attributes.AttributesOrder` only has an issue

**Comment by kukulich** [comment]
@lptn Ok, the error `An error occurred during processing; checking has been aborted. The error message was: Attributes order not specified.` is right and expected.

**Comment by lptn** [comment]
@kukulich 

It seems like we misunderstood each other. When `SlevomatCodingStandard.Attributes.AttributesOrder` in a rule set and failed with `An error occurred during processing; checking has been aborted. The error message was: Attributes order not specified.` error, it breaks the whole sniffing process for the current file (technically - all excluded rules become not excluded and thus the output is full of errors).

**Comment by kukulich** [comment]
Yes, that's expected.

**Comment by lptn** [comment]
ok, I see! This is definitely not the best developer experience, but (I assume) it's not easy to make it better ATM. Thanks again!
